examples/zupass-feed-server/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "license": "GPL-3.0-or-later",
+  "scripts": {
+    "dev": "NODE_ENV=development ts-node-dev --exit-child -- src/main.ts",
+    "build": "tsc --noEmit",
+    "lint": "eslint \"**/*.ts{,x}\"",
+    "start": "NODE_ENV=production PORT=3000 ts-node src/main.ts"
+  },
+  "devDependencies": {
+    "@pcd/eslint-config-custom": "^0.8.0",
+    "@types/circomlibjs": "^0.1.2",
+    "@types/cors": "^2.8.14",
+    "@types/express": "^4.17.17",
+    "@types/json-bigint": "^1.0.2",
+    "@types/jsonfile": "^6.1.3",
+    "@types/libsodium-wrappers-sumo": "^0.7.6",
+    "@types/lodash": "^4.14.199",
+    "@types/pako": "^2.0.1",
+    "@types/qr-image": "^3.2.7",
+    "@types/react": "^18.2.22",
+    "@types/styled-components": "^5.1.27",
+    "@types/uuid": "^9.0.4",
+    "@typescript-eslint/eslint-plugin": "^6.7.2",
+    "eslint": "^8.49.0",
+    "eslint-plugin-react": "^7.33.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-turbo": "^1.10.14",
+    "prettier": "^3.0.3",
+    "ts-node-dev": "^2.0.0",
+    "typescript": "^5.2.2"
+  },
+  "dependencies": {
+    "@pcd/eddsa-frog-pcd": "^0.0.1",
+    "@pcd/eddsa-pcd": "^0.3.0",
+    "@pcd/eddsa-ticket-pcd": "^0.3.0",
+    "@pcd/email-pcd": "^0.3.0",
+    "@pcd/passport-interface": "^0.8.0",
+    "@pcd/pcd-collection": "^0.8.0",
+    "@pcd/pcd-types": "^0.8.0",
+    "@pcd/semaphore-signature-pcd": "^0.8.0",
+    "@types/url-join": "^4.0.2",
```

Comment:
nit: here and in all the other package.json files, it would be nice to be very careful about making sure to not have `@types/` dependencies make it into the `dependencies`, and instead to place them under `devDependencies`.